src/runtime/containerd-shim-v2/service.go

```diff
@@ -27,18 +27,22 @@ import (
 	"github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"go.opentelemetry.io/otel"
-	"go.opentelemetry.io/otel/label"
-	otelTrace "go.opentelemetry.io/otel/trace"
 	"golang.org/x/sys/unix"
 
 	"github.com/kata-containers/kata-containers/src/runtime/pkg/katautils"
+	"github.com/kata-containers/kata-containers/src/runtime/pkg/katautils/katatrace"
 	vc "github.com/kata-containers/kata-containers/src/runtime/virtcontainers"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/compatoci"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/oci"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/types"
 )
 
+// shimTracingTags defines tags for the trace span
+var shimTracingTags = map[string]string{
```

Comment:
Random thought: I'm not 100% sure we want to do this, but we could potentially use the existing logger framework to simplify this PR (_maybe_). For example, most subsystems have a `SetLogger()` or a `logrus.WithFields{}` variable that encodes most of these fields so we could potentially extractc them using `logrus.Entry.Data`, augment them as needed, and then pass the final set of fields to `katatrace.Trace()`. Another advantage of doing this is that the set of logger fields is already added to by the higher level code calling each subsystems `.SetLogger()` function (so we can minimise field duplication).

Reply:
@jodh-intel I think we should consider this, maybe in a different PR?

Reply:
Agreed, I don't think I have the context for what the logs should be. I originally just worked on this as a class project haha. I'll leave that standardization for a different PR.